addons/portal_rating/models/mail_thread.py

```diff
@@ -0,0 +1,10 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class MailThread(models.AbstractModel):
+    _inherit = 'mail.thread'
+
+    def _get_allowed_message_post_params(self):
+        return super()._get_allowed_message_post_params() | {"rating_value"}
```

Comment:
Note: ok for now as it is direct diff related to this PR, but at some point we might want to prevent from setting `rating_value` on unexpected messages (on many documents rating makes no sense, on some it only makes sense at the end of flows, on others it's almost always accepted).